packages/components/src/components/overflow-menu/overflow-menu.hbs

```diff
@@ -5,63 +5,137 @@
   LICENSE file in the root directory of this source tree.
 -->
 
-<div data-overflow-menu tabindex="0" aria-label="Overflow" class="{{@root.prefix}}--overflow-menu" role="button"
-  aria-haspopup="true" aria-expanded="false">
-  {{ carbon-icon 'OverflowMenuVertical16' class=(add @root.prefix '--overflow-menu__icon') }}
-  <ul class="{{@root.prefix}}--overflow-menu-options" tabindex="-1" role="menu" aria-label="Overflow"
-    data-floating-menu-direction="{{direction}}">
-    {{#each items}}
-    <li
-      class="{{@root.prefix}}--overflow-menu-options__option {{#if disabled}} {{@root.prefix}}--overflow-menu-options__option--disabled {{/if}} {{#if danger}} {{@root.prefix}}--overflow-menu-options__option--danger {{/if}}"
-      role="presentation" {{#if disabled}} aria-disabled="true" {{/if}}>
-      <button class="{{@root.prefix}}--overflow-menu-options__btn" role="menuitem" {{#if title}} title="{{title}}"
-        {{/if}} {{#if primaryFocus}} data-floating-menu-primary-focus {{/if}} {{#if disabled}} tabindex="-1" {{/if}}>
-        <div class="{{@root.prefix}}--overflow-menu-options__option-content">
-          {{label}}
-        </div>
-      </button>
-    </li>
-    {{/each}}
-  </ul>
-</div>
+{{!-- @todo remove div with main role once a11y is verified. All demos should have this landmark to lead by example --}}
```

Comment:
We the markup change doesn't require new styles, we can simply replace old markup with new one.

Reply:
I have the todo comment in there because this isn't a good practice to just have main sitting in the middle of the content like this. I'd rather remove it from the demo and add it somewhere where higher up in the preview files to be more realistic to what an offering should be implementing

Reply:
@elizabethsjudd Just to clarify, my comment above is not against `@todo` - Rather than against the newly added markup without replacing the old one. Or did your response allude to such topic?

Reply:
@asudoh I'm not sure what you're asking for in these comments.  That div was a placeholder just for testing AAT. It's been removed at this point. Is there something else you are looking for me to do.